server/WiMicServer.java

```diff
@@ -329,10 +329,13 @@ public static void initialize() throws Exception {
         dataLineInfo = new DataLine.Info(SourceDataLine.class, format);
         sourceDataLine = (SourceDataLine) AudioSystem.getLine(dataLineInfo);
         sourceDataLine.open(format);
-        sourceDataLine.start();
-
-        FloatControl volumeControl = (FloatControl) sourceDataLine.getControl(FloatControl.Type.MASTER_GAIN);
-        volumeControl.setValue(1.00f);
+        
+	sourceDataLine.start();
+        if (sourceDataLine.isControlSupported( FloatControl.Type.MASTER_GAIN)) {
+            FloatControl volumeControl = (FloatControl) sourceDataLine.getControl(
+                                                     FloatControl.Type.MASTER_GAIN);
```

Comment:
This must be properly indented, as well as the closing bracket must be on a new line.
(Refer codebase)